documentation-site/pages/getting-started/installation.mdx

```diff
@@ -25,10 +25,11 @@ npm install baseui styletron-engine-atomic styletron-react
 
 **The next step is to setup Styletron in your application.** There are detailed guides for
 
-* [Fusion.js](https://www.styletron.org/getting-started/#with-fusionjs),
-* [Next.js](https://www.styletron.org/getting-started/#with-nextjs),
-* [Gatsby](https://www.styletron.org/getting-started/#with-gatsby) and
-* [general React applications](https://www.styletron.org/getting-started/#with-react).
+- [Fusion.js](https://www.styletron.org/getting-started/#with-fusionjs),
+- [Next.js](https://www.styletron.org/getting-started/#with-nextjs),
+- [Gatsby](https://www.styletron.org/getting-started/#with-gatsby),
+- [Bedrock](http://t.uber.com/base-bedrock) and
```

Comment:
This is internal only, maybe add it to the internal docs?

Reply:
Is it going to be visible though? We did a similar thing for icons and just added a note [here](https://baseweb.design/components/icon/).

Reply:
in its current form it can be missleading - should we add that it is for uber folks only?

Reply:
@gergelyke Added *Uber only* note